include/common_package/commands.hpp

```diff
@@ -0,0 +1,160 @@
+#pragma once
+
+#include <cinttypes>
+#include <string>
+#include <optional>
+#include <map>
+#include <unordered_set>
+#include <stdexcept>
+#include <nlohmann/json.hpp>
+
+namespace common_lib
+{
+    // Global Defines
+    // TODO adjust with real values
+    const uint16_t MAX_FLIGHT_HEIGHT_CM = 120 /* [m] */ * 100; /// Maximum allowed flight height in cm
```

Comment:
Wissen wir diese Dinge nicht eigentlich schon? Die Anforderungen sind ja fix und was nicht in den Anforderungen steht können wir ja selbst frei festlegen oder?

Reply:
Die können wir komplett frei belegen. Mir sind da keine Anforderungen bekannt.
Hier würde ich in erster Linie eher technische / rechtliche / Sicherheits-Grenzen definieren.
Konkrete Werte können wir dann noch ins Mission Definition File zum Geofence hinzufügen.

Reply:
Ich habe das Mission Definition File (auch mit @J0las Input) entsprechend erweitert: https://docs.google.com/document/d/1G_p5BJzluRZCTSzqP9_-HDcJC5uFrCkSYyJ1a_C1XGY/edit?usp=drive_link